src/mon/MonCommands.h

```diff
@@ -444,6 +444,12 @@ COMMAND("fs rename "
 	"name=new_fs_name,type=CephString,goodchars=" FS_NAME_GOODCHARS
 	" name=yes_i_really_mean_it,type=CephBool,req=false",
 	"rename a ceph file system", "mds", "rw")
+COMMAND("fs swap "
+	"name=fs1_name,type=CephString "
+	"name=fs2_name,type=CephString "
+	"name=swap_fscids,type=CephChoices,strings=yes|no "
```

Comment:
```suggestion
	"name=swap_fscids,type=CephChoices,strings=yes|no,req=true "
```

Reply:
Oh, true. This option should be mandatory. I'll make this change, Thanks!